mppsolar/outputs/hassd_mqtt.py

```diff
@@ -131,10 +133,16 @@ def build_msgs(self, *args, **kwargs):
                 }
                 if device_class:
                     payload["device_class"] = device_class
-                if unit == "W":
-                    payload.update({"state_class": "measurement", "device_class": "power"})
+                #if state_class:
```

Comment:
any reason you didnt go with this approach?

Reply:
I had the problem that my counter in the Home Assistant jumped from time to time, so I had commented on it.
Will test it again.